templates/zerver/emails/password_reset.source.html

```diff
@@ -6,7 +6,7 @@
 
 {% block content %}
     <p>
-        {% trans %}Somebody (possibly you) requested a new password for the Zulip account {{ email }} on {{ realm_uri }}.{% endtrans %}
+        {% trans email=macros.email_tag(email) %}Somebody (possibly you) requested a new password for the Zulip account {{ email }} on {{ realm_uri }}.{% endtrans %}
```

Comment:
Do we actually want this to be a `mailto` tag?  I feel like it might be better framing this as data, not something to click.

Reply:
Slack is linking the email in password reset emails.

![Screenshot from 2020-10-29 20-09-25](https://user-images.githubusercontent.com/7190633/97589048-18f7c280-1a23-11eb-8e97-3b7c0ff001d1.png)

Reply:
Hmm, maybe they're doing it just to make it look visually similar to the URL, but I don't see any purpose in clicking a link to your own email address, so I think we shouldn't do it.